nss-day-cohort-45/hello-world-column-counts
# added main picture; added links and content to each box

## Things Added

1. added main content picture
2. added and linked content for Citizens 
3. added and linked content for Locations
4. added and linked content for Cities
5. updated active nav link

## Testing Tests

1. serve it up
2. make sure main picture of Chairman Mao loads
3. make sure that all links provided link to external sites
4. enjoy learning a few aspects of China
